lib/dynamoid/persistence.rb

```diff
@@ -507,6 +507,11 @@ def persisted?
     # @return [true|false] Whether saving successful or not
     # @since 0.2.0
     def save(options = {})
+      unless Dynamoid.config.create_table_on_save
+        Dynamoid.logger.warn "Table #{table_name} is not created as create_table_enabled is disabled"
+        return false
+      end
+
```

Comment:
Doesn't it prevent model saving at all?

Reply:
Yes, you are right. These unless are hard on the brain.